docs/source/yelpsoa_configs.rst

```diff
@@ -279,6 +279,14 @@ Here is an example smartstack.yaml::
       proxy_port: 20028
       timeout_server_ms: 5000
 
+The main key is the service namespace.  Namespaces were introduced for PaaSTA
+services in order to support running multiple daemons from a single service
+codebase. In PaaSTA, each instance in your marathon.yaml maps to a smartstack
+namespace of the same name, unless you specify a different ``nerve_ns``.
```

Comment:
But... a canary generally shares the main namespace.

I could do another example?